### PR TITLE
update combobox document for version 1.1

### PR DESCRIPTION
**Pull request by gjulivan**
update combobox 1.1

support custom footer and select all button
image
image

### CODE REVIEW

**Comment by gjulivan** [comment]
the widget has been released and this PR can be merged once completed for review

File: content/en/docs/appstore/widgets/combobox.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Combo Box"
```
Comment (ConnorLand) [inline content/en/docs/appstore/widgets/combobox.md]: Hi @gjulivan . Documentation has its own special casing rules. Title nouns need to all be upper-cased. Same with nouns in doc link titles and header titles. Please do not change casings in these instances.

**Review by ConnorLand — APPROVED** [review]

**Comment by ConnorLand** [comment]
Will be merged after 1.1.1 patch release. @gjulivan will notify my via slack.

**Comment by gjulivan** [comment]
https://marketplace.mendix.com/link/component/219304
combobox 1.1.1 released, this PR can be merged
@ConnorLand

**Pull request merged**